src/pages/workspace/expensifyCard/WorkspaceExpensifyCardPage.tsx

```diff
@@ -25,12 +26,17 @@ function WorkspaceExpensifyCardPage({route}: WorkspaceExpensifyCardPageProps) {
 
     const styles = useThemeStyles();
     const theme = useTheme();
-    const [cardSettings] = useOnyx(`${ONYXKEYS.COLLECTION.PRIVATE_EXPENSIFY_CARD_SETTINGS}${workspaceAccountID}`);
-    const [cardsList] = useOnyx(`${ONYXKEYS.COLLECTION.WORKSPACE_CARDS_LIST}${workspaceAccountID}_${CONST.EXPENSIFY_CARD.BANK}`, {selector: filterInactiveCards});
+    const domainCardsID = useDomainCardsID(policyID);
+
+    // TODO: add logic for choosing between the domain and workspace feed when both available
+    // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
+    const cardsID = domainCardsID || workspaceAccountID;
```

Comment:
Maybe we can call this domainAccountID since that's what it is in the db. And technically a workspaceAccountID is a type of domainAccountID. Or maybe we call it the cardAccountID.

Reply:
I think `cardsID` is easily mixed up with `cardID`.